index.html

```diff
@@ -17,123 +17,131 @@
-                    <path class="svg-element" d="M17.4543 29.307L16.3935 30.3678C16.2888 30.4722 16.2056 30.5961 16.1488 30.7326C16.092 30.8691 16.0627 31.0155 16.0625 31.1634C16.0623 31.3112 16.0913 31.4577 16.1479 31.5943C16.2044 31.7309 16.2873 31.8551 16.3918 31.9596C16.4964 32.0642 16.6205 32.1471 16.7571 32.2036C16.8938 32.2601 17.0402 32.2891 17.1881 32.2889C17.3359 32.2888 17.4823 32.2595 17.6188 32.2026C17.7553 32.1458 17.8793 32.0627 17.9836 31.9579L19.0444 30.8971C19.1491 30.7928 19.2322 30.6688 19.289 30.5323C19.3457 30.3958 19.375 30.2495 19.3751 30.1017C19.3752 29.9538 19.3462 29.8075 19.2897 29.6709C19.2332 29.5343 19.1503 29.4102 19.0458 29.3056C18.9413 29.2011 18.8172 29.1182 18.6806 29.0617C18.544 29.0052 18.3976 28.9762 18.2498 28.9763C18.1019 28.9764 17.9556 29.0057 17.8191 29.0625C17.6826 29.1192 17.5587 29.2023 17.4543 29.307Z"/>
-                    <path class="svg-element" d="M17.4543 18.697C17.5587 18.8016 17.6827 18.8845 17.8191 18.9412C17.9556 18.9978 18.1019 19.027 18.2496 19.027C18.3973 19.0271 18.5436 18.998 18.6801 18.9415C18.8166 18.885 18.9406 18.8022 19.0451 18.6977C19.1496 18.5932 19.2324 18.4692 19.2889 18.3327C19.3454 18.1962 19.3745 18.0499 19.3744 17.9022C19.3744 17.7544 19.3452 17.6082 19.2886 17.4717C19.2319 17.3353 19.149 17.2113 19.0444 17.1069L17.9836 16.0461C17.8793 15.9413 17.7554 15.858 17.6188 15.8011C17.4823 15.7442 17.3359 15.7148 17.1879 15.7146C17.04 15.7144 16.8935 15.7434 16.7568 15.7999C16.6201 15.8564 16.4959 15.9393 16.3913 16.0439C16.2867 16.1485 16.2038 16.2727 16.1473 16.4094C16.0908 16.5461 16.0618 16.6926 16.062 16.8405C16.0622 16.9885 16.0916 17.1349 16.1485 17.2714C16.2054 17.408 16.2887 17.5319 16.3936 17.6362L17.4543 18.697Z"/>
-                </svg>               
-            </label>
-
-            <a href="https://discord.com" target="_blank" class="button-action">  
+                <div class="page-header-buttons">
+                <input type="checkbox" id="themeCheckbox" class="theme-checkbox hidden">
+
+                <label for="themeCheckbox" class="theme-label">
+                    <svg width="48" height="48" viewBox="0 0 48 48" class="svg-icon">
```

Comment:
I think would be better to move markup of this icon into sprite as well. If you do this, user will download markup of it only once during the first page open. If not, user will download it as a part of each page.

Also, please move logo.svg and wallet.svg into sprite as well.

💚 3.7 Use vector sprites where applicable

Reply:
Liza again 😄 :
she said that icons that have animation (and here we have colour changing) are better added as <svg> on <html>

Reply:
Well. There's some restrictions from browsers:
You can't animate `fill` property of svg if it's in <img> or other replacing tag (object, embed, iframe).
But if it's used as SVG, don't matter if it's an inline usage or sprite, changing a `fill` property will work correctly, even with transition.
So I don't see any reason to inline this icon here, because sprites are applicable.